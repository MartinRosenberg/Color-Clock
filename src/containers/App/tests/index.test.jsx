import { shallow } from 'enzyme'
import React from 'react'

import { App } from '..'

describe('App container', () => {
  beforeEach(() => {
    // Moment uses `Date` internally, so overwriting `Date.now` is a simple mock
    Date.now = jest.fn(() => 0)
  })

  test('should render correctly', () => {
    const wrapper = shallow(<App/>)
    expect(wrapper).toMatchSnapshot()
  })
})
