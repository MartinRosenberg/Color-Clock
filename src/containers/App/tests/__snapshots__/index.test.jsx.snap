// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App container should render correctly 1`] = `
<Fragment>
  <Body
    color="#180000"
  />
  <Clock
    color="#180000"
    time="18:00:00"
  />
</Fragment>
`;
